test/server/generators.ts

```diff
@@ -0,0 +1,51 @@
+import { expect, test } from 'vitest'
+import { app } from './utils'
+
+test('typescript generator route', async () => {
+  const res = await app.inject({
+    method: 'GET',
+    path: '/generators/typescript',
+  })
+  expect(res.statusCode).toBe(200)
+  expect(res.headers['content-type']).toContain('text/plain')
+  expect(res.body).contain('public')
+})
+
+test('go generator route', async () => {
+  const res = await app.inject({
+    method: 'GET',
+    path: '/generators/go',
+  })
+  expect(res.statusCode).toBe(200)
+  expect(res.headers['content-type']).toContain('text/plain')
+  expect(res.body).toBeTruthy()
+})
+
+test('swift generator route', async () => {
+  const res = await app.inject({
+    method: 'GET',
+    path: '/generators/swift',
+  })
+  expect(res.statusCode).toBe(200)
+  expect(res.headers['content-type']).toContain('text/plain')
+  expect(res.body).toBeTruthy()
+})
+
+test('generator routes with includedSchemas parameter', async () => {
```

Comment:
valid, but a lot of actual issues in the code may be missed, would be better to include one or a few of multiple existing schemas and check that only they are included